### PR TITLE
[#FLAT-223-225] Submit recipe using normal discourse popup window

### PR DESCRIPTION
**Pull request by ArthurZaharov**
https://sourceclear.atlassian.net/browse/FLAT-223
https://sourceclear.atlassian.net/browse/FLAT-225

### CODE REVIEW

File: assets/javascripts/discourse/routes/app-route-map.js.es6 [hunk 1]
```diff
@@ -10,8 +10,7 @@ export default function() {
   });
   this.resource('contact', { path: '/contact' });
   this.resource('about-site', { path: '/about-site' });
-  this.resource('recipes', { path: '/recipes' });
-  this.resource('recipesNew', { path: '/recipes/new' });
+  this.resource('recipes', { path: 'c/recipes' });
```
Comment (AskarZinurov) [inline assets/javascripts/discourse/routes/app-route-map.js.es6]: this.resource('recipes', { path: '/c/recipes' }); ?

Comment (ArthurZaharov) [inline assets/javascripts/discourse/routes/app-route-map.js.es6]: I replace discourse default path for category (aka recipes) to our new resource (this give us ability to template etc.)

File: app/controllers/discourse_reports/recipes_controller.rb [hunk 2]
```diff
@@ -15,33 +17,14 @@ def index
...
-
     private
 
-    def topic
-      @post ||= topics.includes(:posts).find(params[:id])
-    end
-
     def topics
-      Topic
+      topics = Topic
```
Comment (AskarZinurov) [inline app/controllers/discourse_reports/recipes_controller.rb]: Seems little bit confusing when your local variable name same as function name.

**Comment by AskarZinurov** [comment]
:ok:

**Pull request merged**